tasks/orca-recovery-adapter/schemas/output.json

```diff
@@ -1,82 +1,77 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
so we don't need to specify the schema version?

Reply:
no. It's from original [orca schema](https://github.com/nasa/cumulus-orca/blob/v8.0.1/tasks/orca_recovery_adapter/schemas/output.json#L2)